nucleus/packager/external/bean-validator-cdi/pom.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+    Copyright (c) 1997-2017 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This is a packaging module copied from hk2 as HK2 team does not support this any more and want GlassFish modules to take care of it. So, this is not a new file.